traffic_portal/app/src/common/modules/table/agGrid/CommonGridController.js

```diff
@@ -217,6 +230,9 @@ let CommonGridController = function ($scope, $document, $state, userModel, dateU
                 $scope.$apply();
             },
             onRowClicked: function(params) {
+                if (params.event.target.classList.contains('link')) {
+                    return;
+                }
```

Comment:
shouldn't this just check `params.event.target instanceof HTMLAnchorElement` ? Classes can be improperly applied or overloaded for styling reasons, seems more reliable to just directly check if the target is actually a link.

Reply:
Probably, I just copied it from the servers page